modules/sagemaker/sagemaker-endpoint/README.md

```diff
@@ -31,7 +31,10 @@ where endpoints are provisioned as part of automated Continuous Integration and
 
 - `sagemaker-project-id`: SageMaker project id
 - `sagemaker-project-name`: SageMaker project name
+- `sagemaker-domain-id`: SageMaker domain id
+- `sagemaker-domain-arn`: SageMaker domain ARN. Used for domain resource isolation.
```

Comment:
This description makes it sound like this parameter is required if `enable_isolation` is true. Can you add a brief explanation for how this parameter is used?

Reply:
Have added some more detail to how the parameter is used in new commit, let me know if you think it needs any more detail